common/emotes.go

```diff
@@ -17,19 +17,11 @@ var (
 	reWrappedEmotes = regexp.MustCompile(`[:\[][^\s:\/\\\?=#\]\[]+[:\]]`)
 )
 
-func init() {
-	Emotes = NewEmotesMap()
-}
-
-func NewEmotesMap() EmotesMap {
-	return map[string]string{}
-}
-
 func (em EmotesMap) Add(fullpath string) EmotesMap {
 	fullpath = reStripStatic.ReplaceAllLiteralString(fullpath, "")
 
-	base := filepath.Base(fullpath)
-	code := base[0 : len(base)-len(filepath.Ext(base))]
+	base := path.Base(fullpath)
+	code := base[0 : len(base)-len(path.Ext(base))]
```

Comment:
We might want to still use `filepath.Ext()` here, as `path.Ext()` always assumes the path separator is `/`, where `filepath.Ext()` will use the OS's path separator.

Reply:
fixed in df74c677ff2e18b3ba732667c03a9c0f89430861